### DISCUSSION TITLE
✅ Feature request Add Dynamic URLs for Internal Linking

### DISCUSSION THREAD

**Discussion by orionp13** [elementor/elementor#23687] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I change the url of my page and the link is broken

### Describe the solution you'd like

I'd like the urls to change dynamically like on divi if I change the url name I don't need to manually change the names 

On divi they describe the feature
https://www.elegantthemes.com/blog/divi-resources/how-to-add-dynamic-urls-to-your-ctas-with-divi-acf

It can be great to get the same

**Comment by nicholaszein** [comment]
Hello @orionp13!

This feature is already available with Elementor Pro's [Dynamic Tags](https://elementor.com/help/dynamic-tags-pro-2/), introduced 6 years ago in Elementor Pro version 2.0.0.

In your case you can use the option **`Internal URL`** from the **`Site`** list in the dynamic tag. Here's the [documentation](https://elementor.com/help/dynamic-tags-pro-2/#site).

Please, before submitting your suggestions, make sure to visit the Help Center to check if the feature you are requesting isn't already available.

I hope this helps.

I'm closing this request as solved.

Kind regards
